src/Components/ImagesTable/ClonesTable.tsx

```diff
@@ -41,7 +41,7 @@ const Ami = ({ status }: AmiPropTypes) => {
       return undefined;
 
     default:
-      return <Skeleton width="12rem" />;
+      return;
```

Comment:
Nice! The code could be cleaned up a bit more. As we expect to get an AMI for success, and undefined for failure and by default we could use:

```javascript
    case 'failure':
    default:
      return undefined;
  }
```

or even 
```javascript
    default:
      return undefined;
  }
```

if we don't mind including failure with all the other cases when we don't display AMI. What do you think?

Reply:
good idea thank you @regexowl